papis/arxiv.py

```diff
@@ -73,7 +75,7 @@ def get_data(
     url = urllib.request.Request(
         req_url,
         headers={
-            'User-Agent': papis.config.get('user-agent')
+            'User-Agent': papis.config.get('user-agent')  # type: ignore[dict-item]
```

Comment:
Not sure ignoring the type warning is the best idea here. The issue seems to be that `get` returns an `Optional[Any]` but `Request` expects a string.